### PR TITLE
Update README - Closes #1432

### LINKED ISSUE #1432: Update README
### Which content is outdated?
The README is describing an outdated structure of the `lisk-docs` repository.

### What should be the new content?
Links to pages of the docs manual should be linked here for further information.
For example, about getting started, contributing, local preview, see also #1425

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1432

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,34 +1,98 @@
...
-Click here to see the live version of the https://lisk.io/documentation/lisk-sdk/index.html[Lisk Documentation].
+== Structure
 
-xref:CONTRIBUTING.adoc[How to contribute to the Lisk Documentation]
+The internal structure of `development` and `main` branch.
 
-== Overview
+=== The `docs/` folder
+
+The source content for the general Lisk documentation  is located in the `docs/` folder.
```
Comment (chris529) [inline README.adoc]:
```suggestion
The source content for the general Lisk documentation is located in the `docs/` folder.
```

File: README.adoc [hunk 2]
```diff
@@ -1,34 +1,98 @@
...
+
+=== The `ui/` folder
+
+The presentation of the documentation is stored completely separate from the actual content.
+
+The UI is based on the {url_antora_ui}[Antora default UI^] (see also {url_antora_ui_docs}[^]).
+
+It is customized as listed in the following points below:
+
+* Updated the CSS to follow improve UX and follow Lisk brand styleguide.
```
Comment (chris529) [inline README.adoc]:
```suggestion
* Updated the CSS to improve the UX and follow the Lisk brand styleguide.
```

File: README.adoc [hunk 3]
```diff
@@ -1,34 +1,98 @@
...
+* Added additional features, such as:
+** Algolia site search
+** Dark theme
+** Zoom for images
+** Additional layout for swagger UI
+** Tab-blocks
+** ...
+
+=== The `build/` folder
+The `build` folder contains...
```
Comment (chris529) [inline README.adoc]:
```suggestion
The `build` folder contains the following:
```

File: README.adoc [hunk 4]
```diff
@@ -1,34 +1,98 @@
...
+
+* ...the `site.yml` file, which is used to generate the Lisk documentation using Antora.
+* ...the `searchdocs-scraper` folder, for creating the search index.
+* ...the `lib` folder, which stores the Asciidoc extensions, that are used in the Lisk documentation.
```
Comment (chris529) [inline README.adoc]:
```suggestion
* ...the `lib` folder which stores the Asciidoc extensions that are used in the Lisk documentation.
```

File: README.adoc [hunk 5]
```diff
@@ -1,34 +1,98 @@
...
+<4> The playbook used to compile the documentation at {url_docs_gitpage}[^] with Antora.
+Results are pushed to the `git-page` branch.
+
+== Product-specific docs
+
+There are additional content sources, which each store the documentation for a specific software product of Lisk.
+
+=== Content sources
+
+NOTE: These branches store the latest or WIP versions of each component.
```
Comment (chris529) [inline README.adoc]:
```suggestion
NOTE: These branches store either the latest or the WIP versions of each component.
```

File: README.adoc [hunk 6]
```diff
@@ -1,34 +1,98 @@
...
-docs-sdk-VERSION
-docs-protocol-VERSION
-----
-where `VERSION` is always referring to the corresponding software version.
+where
 
-UI::
-The presentation of the documentation is stored completely seperate form the actual content.
-It's stored on the `antora` branch.
+** `COMPONENT` can be either `sdk`, `core` or `service`.
```
Comment (chris529) [inline README.adoc]:
```suggestion
** `COMPONENT` can be either `sdk`, `core`, or `service`.
```

**Review by chris529 — COMMENTED** [review]
Looks good, just a few minor points 👍

**Review by shuse2 — APPROVED** [review]

**Pull request merged**